Sources/Private/CoreAnimation/Animations/GradientAnimations.swift

```diff
@@ -53,6 +53,8 @@ extension GradientRenderLayer {
       },
       context: context)
 
+    try addOpacityAnimation(for: gradient.opacity, context: context)
```

Comment:
While I was working on this I also noticed we were ignoring the opacity configuration on `GradientFill`s, so I added that here like we did for `Fill`s in #1589